easybuild/easyconfigs/p/paladin/paladin-1.4.6-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,41 @@
+
+easyblock = 'MakeCp'
+
+name = 'paladin'
+version = '1.4.6'
+
+homepage = 'https://github.com/ToniWestbrook/paladin'
+description = """
+Protein ALignment And Detection INterface
+
+PALADIN is a protein sequence alignment tool designed for the accurate functional characterization of metagenomes.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ToniWestbrook/paladin/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['fd23a8907d832cada060e112addb1b3b3751353dda515887c69c1772fd37cc4a']
+
+builddependencies = [
+    ('binutils', '2.37'),
+]
+
+# Use EB build flags
+buildopts = 'CFLAGS="$CFLAGS"'
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('cURL', '7.78.0'),
+    ('OpenSSL', '1.1', '', True),
+]
+
+files_to_copy = [(['paladin'], 'bin'), 'run.py', 'sample_data', 'scripts']
+
+sanity_check_paths = {
+    'files': ['bin/paladin'],
+    'dirs': ['sample_data'],
+}
```

Comment:
@verdurin
can you add a `sanity_check_commands` here? for example
```
sanity_check_commands = ['paladin --help']
```

Reply:
@smoors now added